l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -125,9 +125,10 @@ def getPartnerBase(self, DatiAnagrafici, supplier=True):
                 )
         partners = partner_model
         if vat:
-            partners = partner_model.search([
-                ('vat', '=', vat),
-            ])
+            domain = [('vat', '=', vat)]
+            if cf:
+                domain.append(('fiscalcode', '=', cf))
+            partners = partner_model.search(domain)
```

Comment:
In questo modo però se a sistema ho un fornitore con solamente la P.IVA e nell'XML quel fornitore inserisce anche il CF, verranno creati 2 partner con la stessa partita IVA, giusto?

Comunque, in generale, secondo me bisogna cambiare approccio, in considerazione anche di https://github.com/OCA/l10n-italy/issues/1667#issuecomment-594459593

Reply:
La mia era un'idea senza casi concreti nè casistica completa, ho scritto di là